Casks/d/devonsphere.rb

```diff
@@ -0,0 +1,38 @@
+cask "devonsphere" do
+  on_catalina :or_older do
+    version "1.9.7"
+    sha256 "2240a88c65f6de66b554ea802030effb0fa88ba6ab41fca84af3e0b5afdffbf1"
+
+    livecheck do
+      skip "Legacy version"
+    end
+  end
+  on_big_sur :or_newer do
+    version "1.9.8"
+    sha256 "a9bae928bf58c1a5cbb11878c306ec1e7a3d2bdc5de3e5d080f291c5363c6997"
+
+    livecheck do
+      url "https://api.devontechnologies.com/2/apps/sparkle/sparkle.php?id=900000028"
+      strategy :sparkle do |items|
+        items.map(&:version)
+      end
```

Comment:
```suggestion
      strategy :sparkle
```

I think the default should work here, unless I'm missing something.